modules/oadp-timeouts.adoc

```diff
@@ -0,0 +1,194 @@
+----
+
+.Procedure
+* Edit the values in the `spec.configuration.restic.timeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  configuration:
+    restic:
+      timeout: 1h
+  ...
+----
+
+[id="Velero-resource-timeout_{context}"]
+== Velereo resource timeout
+
+.Description
+`resourceTimeout` defines how long to wait for several Velero resources before timeout occurs, such as Velero CRD availability, `volumeSnapshot` deletion, and repo availability. The default is 10m.
+
+.Implementation
+.. Use this timeout for backups with very large volumes . This parameter is used as a timeout value when Velero tries to clean-up or delete the Container Storage Interface (CSI) snapshots, before marking the backup as complete.
+... A sub-task of this cleanup tries to patch VSC and this timeout can be used for that task.
++
+.. Use this timeout to create or ensure a backup repository is ready for filesystem based backups for Restic or Kopia.
+.. Use this timeout to check if the Velero CRD is available in the cluster before restoring the custom resource (CR) or resource from the backup.
+
+.Procedure
+* Edit the values in the `spec.configuration.velero.resourceTimeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  configuration:
+    velero:
+      resourceTimeout: 10m
+  ...
+----
+
+[id="Datamover-timeout_{context}"]
+== Data Mover timeout
+
+.Description
+`timeout` is a user supplied timeout to be used for `VolumeSnapshotBackup` and `VolumeSnapshotRestore` to complete. The default value is 10m.
+
+.Implementation
+.. Use this timeout when VSB/VSR creation times out after 10min, use this setting to adjust the timeout appropriately.
+.. For large scale environments set timeout for 1hr.
+.. Use this timeout with the vsm plugin.
+.. Use this timeout with OADP 1.1.x.
+
+.Procedure
+* Edit the values in the `spec.features.dataMover.timeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  features:
+    dataMover:
+      timeout: 10m
+  ...
+----
+
+[id="CSIsnapshot-timeout_{context}"]
+== CSI snapshot timeout
+
+.Description
+`CSISnapshotTimeout` specifies the time used to wait for `CSI VolumeSnapshot`, the status turns to `ReadyToUse` during creation, before returning error as timeout. The default value is 10m.
+
+.Implementation
+.. Use this timeout with the csi plugin.
+.. Very large volumes may take longer than 10 minutes to snapshot, adjust this timeout if timeouts are found in the logs.
+.. It is unlikely that this timeout will require adjusting.
```

Comment:
This sounds like more of a note. How about:

[NOTE]
====
Typically, the default value for `CSISnapshotTimeout` does not require adjustment, because the default setting can accommodate large storage volumes. 
====

Reply:
I like your rewording, but I don't want to add a note. I'll replace the text with your text.